src/main/resources/META-INF/reaktivity/core.idl

```diff
@@ -62,6 +63,29 @@ scope core
         struct Unrouted extends core::control::Frame [0x40000002]
         {
         }
+
+        scope auth
```

Comment:
I put this in the same idl file rather than a different one as we had discussed to make it easier to make sure we don't accidentally create duplicate frame id's